src/content/1.0.0/library/nodejs/nodejs-overview.md

```diff
@@ -19,6 +19,7 @@ New Ledger | `new_ledger` | Creates a new ledger
 Predicate ID | `predicate_id` | Returns the ID of a predicate
 Predicate Name | `predicate_name` | Returns the name of a predicate
 Resolve Ledger | `resolve_ledger` | Resolves a ledger identity in the current connection
+Search | `search` | Executes a search against a given a db (ledger instance) and flake parts
```

Comment:
"a given db" instead of "a given a db"

Reply:
Thx - will check in with next set of changes!